packages/bootstrap-4/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -13,36 +12,42 @@ const CheckboxWidget = (props: WidgetProps) => {
     disabled,
     readonly,
     label,
+    schema,
     autofocus,
     onChange,
     onBlur,
     onFocus,
   } = props;
 
-  const _onChange = ({}, checked: boolean) => onChange(checked);
+  const _onChange = ({
+    target: { checked },
+  }: React.FocusEvent<HTMLInputElement>) => onChange(checked);
   const _onBlur = ({
-    target: { value },
-  }: React.FocusEvent<HTMLButtonElement>) => onBlur(id, value);
+    target: { checked },
+  }: React.FocusEvent<HTMLInputElement>) => onBlur(id, checked);
   const _onFocus = ({
-    target: { value },
-  }: React.FocusEvent<HTMLButtonElement>) => onFocus(id, value);
+    target: { checked },
+  }: React.FocusEvent<HTMLInputElement>) => onFocus(id, checked);
 
   return (
-    <FormControlLabel
-      control={
-        <Checkbox
+    <div className={`checkbox ${disabled || readonly ? "disabled" : ""}`}>
+      {(label || schema.description) && (
```

Comment:
Minor nit, not required to change, but I'd store a `const description = label || schema.description` before rendering to avoid having to do the check twice -- for the benefit of future reader/editor, who might accidentally change one line but not the other, leading to unexpected effects

Reply:
make sense, thanks I will change it.

Reply:
https://github.com/MLH-Fellowship/react-jsonschema-form/issues/74